nrcemt/qeels/engine/peak_detection.py

```diff
@@ -191,37 +188,37 @@ def bulk_calculations(peak_position_x, peak_position_y, spectrogram):
               int(e_bulk+max_index_y)-1:int(e_bulk+max_index_y)+1] = 10000
 
     # e dispersion is equalt to e_pixel
-    e_dispersion = BULK_EV/e_bulk
+    e_dispersion = bulk_ev/e_bulk
     return e_dispersion, image
 
 
 def calculation_q(
-    q_pixel, peak_position_x, peak_position_y, e_pixel
+    q_pixel, peak_position_x, peak_position_y, e_pixel, omega
 ):
     """ function calculates the sum of squares based
     on the passed q and e pixels and the peak positions """
-    yfit = calculate_yfit(q_pixel, peak_position_y)
+    yfit = calculate_yfit(q_pixel, peak_position_y, omega)
     SSres = np.sum((peak_position_x*e_pixel - yfit)**2)
     return SSres
 
 
-def calculate_yfit(q_pixel, peak_position_y):
+def calculate_yfit(q_pixel, peak_position_y, omega):
     yfit = (
-        ((OMEGA**2)/2 + (SPEED_LIGHT*q_pixel*peak_position_y)**2 -
-            ((OMEGA**4)/4 + (SPEED_LIGHT*q_pixel*peak_position_y)**4)
+        abs((omega**2)/2 + (SPEED_LIGHT*q_pixel*peak_position_y)**2 -
```

Comment:
was this abs missing before?

Reply:
it isnt supposed to be there, i was messing around with some stuff. In theory it should be the same result

Reply:
if it isn't supposed to be there should we remove it?

Reply:
ive removed it

Reply:
forgot to push?

Reply:
i was just going to include it with the tkinter changes. but its putting up more of a fight than i thought it would.

Reply:
I'm very confused